taxi/models.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True, null=True)
+
+    class Meta:
+        ordering = ("username",)
+
+
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer, on_delete=models.CASCADE, related_name="cars"
+    )
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL, related_name="cars")

```

Comment:
You're using `settings.AUTH_USER_MODEL` in your `ManyToManyField`, which is a best practice in Django. This allows you to switch out the user model if needed in the future. Good job on this! This is a code quality issue.